targets/ESP32/ESP32_C61/nanoCLR/System.Device.Pwm/sys_dev_pwm_native_System_Device_Pwm_PwmChannel.cpp

```
@@ -0,0 +1,415 @@
+//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include "sys_dev_pwm_native.h"
+#include <esp32_os.h>
+#include "Esp32_DeviceMapping.h"
+
+// Used to map a PWM channel number to a pin number for High and low speed channels
+static char HighSpeedPinMap[8] = {255, 255, 255, 255, 255, 255, 255, 255};
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)